modules/rosa-upgrading-4_7_to_4_8_prerequisites.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * rosa_upgrading/rosa-upgrading-sts.adoc
+
+[id="rosa-upgrading-4-7-to-4-8-prerequisites_{context}"]
+= Prerequisites for upgrading from 4.7 to 4.8
```

Comment:
Because this information includes a procedure, consider "Preparing to upgrade from 4.7 to 4.8".